zeebe/gateway-rest/src/test/java/io/camunda/zeebe/gateway/rest/controller/IncidentControllerTest.java

```diff
@@ -43,6 +46,26 @@ public class IncidentControllerTest extends RestControllerTest {
   static final String INCIDENT_SEARCH_URL = INCIDENT_BASE_URL + "/search";
   static final String INCIDENT_PROCESS_INSTANCE_STATISTICS_URL =
       INCIDENT_BASE_URL + "/statistics/process-instances";
+  static final String INCIDENT_STATISTICS_URL =
```

Comment:
🙃 I guess that constant's name could be more precise, maybe. It's process instance statistics by definition, I guess. 🙂